src/lib/storage/reference_segment.hpp

```diff
@@ -32,7 +32,7 @@ class ReferenceSegment : public AbstractSegment {
 
   ColumnID referenced_column_id() const;
 
-  std::shared_ptr<AbstractSegment> copy_using_memory_resource(MemoryResource* /*memory_resource*/) const override;
```

Comment:
When the context is clear (what is the case here), we actually prefer `/*memory_resource*/`.